yang/vyatta-policy-route-v1.yang

```diff
@@ -972,6 +972,13 @@ module vyatta-policy-route-v1 {
 							description "IS-IS level";
 							configd:help "IS-IS level";
 						}
+						leaf src {
+							type union {
```

Comment:
There is an existing type definition for this in [vyatta-types-v1-yang](https://github.com/danos/vyatta-cfg/blob/1dd17fe933c8ad9f72c9f64d884634435bf30a98/yang/vyatta-types-v1.yang#L109). The advantage of the existing type is that it also ensures values are normalized into a canonical form.

vyatta-types-v1 is already imported, so you can use the type like so:
```
type types:ip-address
```